ji_prospector/prospector/migrations/0021_auto_20200626_0842.py

```diff
@@ -0,0 +1,149 @@
+# Generated by Django 3.0 on 2020-06-26 08:42
```

Comment:
Idéalement il faudrait merger les migrations, mais bon c'est pas trop grave :P

Reply:
Ah je me suis demandé si c'était possible, comment on fait ? Ou alors j'aurais dû le faire avant ?

Reply:
https://stackoverflow.com/questions/40028586/how-to-squash-recent-django-migrations

Reply:
J'ai fait ça mais ils disent que c'est mieux de les laisser jusqu'à ce qu'on ait migré partout, donc je suis pas sûre que ça ait simplifié quoi que ce soit :')

Reply:
Si si, en général ce conseil est valide quand tu as plusieurs prod pas au même commit, sauf que là on en a qu'une.
Et en plus elle est sur master, donc merger les migrations de ta branche n'a aucun effet, vu qu'elles ne sont jamais appliquées à la prod ^^